torch2trt/torch2trt.py

```diff
@@ -426,6 +426,8 @@ def forward(self, *inputs):
             bindings[idx] = output.data_ptr()
 
         for i, input_name in enumerate(self.input_names):
+            if not inputs[i].is_contiguous():
+                raise ValueError("Input %s is not contiguous" % input_name)
```

Comment:
Do you see any downside in the following?

```python
if not inputs[i].is_contiguous():
    inputs[i] = inputs[i].contiguous()
```

This may match the original PyTorch model's behavior better.

Reply:
That sounds fine to me as well.  I'll change to that.

Reply:
Note that `.contiguous()` is no-op if tensor is already contiguous.